android/src/main/kotlin/com/hypertrack/sdk/flutter/common/HyperTrackSdkWrapper.kt

```diff
@@ -0,0 +1,246 @@
+package com.hypertrack.sdk.flutter.common
+
+import com.hypertrack.sdk.*
+import java.lang.IllegalStateException
+import java.lang.RuntimeException
+
+
+/**
+ * This class store SDK instance, calls HyperTrack SDK methods and serializes responses
+ */
```

Comment:
As it's a public one, it deserves a better javadoc,
At least

```
Class stores SDK instance, calls HyperTrack SDK methods and serializes responses.
```

We need more docs for the methods

Reply:
It's private one (I will fix the access modifiers)

Reply:
Well, it does not mean that we shouldn't have javadocs for the private ones

Reply:
fixed